src/Core/Silk.NET.Core/Contexts/MultiNativeContext.cs

```diff
@@ -42,6 +42,21 @@ public nint GetProcAddress(string proc, int? slot = default)
             return default;
         }
 
+        /// <inheritdoc />
+        public bool TryGetProcAddress(string proc, out nint addr, int? slot = default)
+        {
+            addr = 0;
+            foreach (var nativeContext in Contexts)
+            {
+                if (nativeContext?.TryGetProcAddress(proc, out addr, slot) ?? false)
+                {
+                    return true;
+                }
+            }
+            
+            return false;
```

Comment:
consider zeroing the addr at the end - just to be consistent in all and every case.

Reply:
Good shout!